masterlist.yaml

```
@@ -21732,3 +21732,20 @@ plugins:
     msg: [ *alreadyInOrFixedByBDSv3_0 ]
 
 ###### Anything new can go after this (If you're not sure where to put it) ######
+
+  # Expanded Tel Mithryn
+  - name: 'telmith.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/18956' ]
```

Instead of a comment, the name of the mod can be added to the location:

```suggestion
  - name: 'telmith.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/18956/'
        name: 'Expanded Tel Mithryn'
```